Formula/utftex.rb

```diff
@@ -0,0 +1,21 @@
+class Utftex < Formula
+  desc "Pretty print math in monospace fonts, using a TeX-like syntax"
+  homepage "https://github.com/bartp5/libtexprintf"
+  url "https://github.com/bartp5/libtexprintf/archive/refs/tags/v1.15.tar.gz"
+  sha256 "a1ecfe44f396c77565515ee1d93207e794d3b99f3423fd2b8e32d53ef59c9d50"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "./autogen.sh"
+    system "./configure", *std_configure_args, "--disable-silent-rules", "--prefix=#{prefix}"
```

Comment:
This is in the standard arguments
```suggestion
    system "./configure", *std_configure_args, "--disable-silent-rules"
```